Userland/Applications/PixelPaint/FilterParams.h

```diff
@@ -101,7 +101,7 @@ class GenericConvolutionFilterInputDialog : public GUI::Dialog {
 
 template<size_t N>
 struct FilterParameters<Gfx::SpatialGaussianBlurFilter<N>> {
-    static OwnPtr<typename Gfx::SpatialGaussianBlurFilter<N>::Parameters> get()
+    static OwnPtr<typename Gfx::SpatialGaussianBlurFilter<N>::Parameters> get(bool should_wrap = true)
```

Comment:
I agree with you. Didn't do it before because I thought the maintainers wouldn't like me messing with it before there were more parameters already. 
But since you suggested I did it, creating the struct `Gfx::ConvolutionFilterOptions`. I don't know if this was what you had in mind, and if i did it in the correct way, so if you have any suggestion please tell me.